flips/20220715-cadence-purity-analysis.md

```diff
@@ -0,0 +1,211 @@
+# Function Purity 
```

Comment:
Given that the proposal is not about pure functions anymore, maybe rename this proposal / language feature to "View functions", [similar to Solidity](https://docs.soliditylang.org/en/v0.8.13/contracts.html#view-functions).